src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/client/PaymentClient.java

```diff
@@ -10,7 +10,8 @@
 import org.springframework.web.bind.annotation.RequestHeader;
 import org.springframework.web.bind.annotation.RequestParam;
 import uk.gov.hmcts.reform.finrem.caseorchestration.model.fee.FeeResponse;
-import uk.gov.hmcts.reform.finrem.caseorchestration.model.pba.payment.PaymentRequest;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.pba.payment.PaymentRequestWithCaseType;
```

Comment:
If one of the payment request types is going to be withdrawn in the future, I would make the changes to the existing and used class, ie. `PaymentRequest` and rename the class to be removed in the future to `PaymentRequestWithSiteID` like was done here.

Reply:
Good shout, I have updated the "New" value to be just `PaymentRequest` and I have renamed the old value to be "PaymentRequestWithSiteId" for when it needs to be depreciated 👍